.github/workflows/build-release.yaml

```
@@ -9,24 +9,22 @@ jobs:
     name: Release Go Binary
     runs-on: ubuntu-latest
     steps:
-      # See https://github.com/onflow/flow-cli/pull/1431 for more information
-      - name: Free up disk space
-        run: |
-          sudo rm -rf ${RUNNER_TOOL_CACHE}
-          sudo rm -rf /usr/share/dotnet
-          sudo rm -rf /usr/local/share/chromium
-          sudo rm -rf /usr/local/share/powershell
-          sudo rm -rf /usr/local/share/vcpkg
-          sudo rm -rf /usr/local/share/miniconda
-          sudo rm -rf /opt/ghc
-          sudo rm -rf /opt/hostedtoolcache/CodeQL
-          sudo rm -rf /usr/local/share/boost
-          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
-          sudo docker image prune --all --force
-          sudo docker builder prune -a -f
-          sudo apt-get clean
-          df -h
       - uses: actions/checkout@v6
+        with:
+          fetch-depth: 0          # ensure tags/history for goreleaser
+          fetch-tags: true
+
+      - name: Free disk space (keep toolcache)
+        uses: jlumbroso/free-disk-space@main
```

```suggestion
        uses: jlumbroso/free-disk-space@v1.3.1
```

will pin version for security